### ISSUE TITLE
Training script error in 02_getting_started_tensorflow/sagemaker-notebook.ipynb

### ISSUE THREAD

**Issue by NickWilkinson37** [huggingface/notebooks#91]
When the `train.py` script is used as is, the model doesn't train successfully on a SageMaker notebook instance, when using the built in `conda_tensorflow2_p36` conda environment. This seems to be due to the dataset not being shuffled. The model always outputs `LABEL_0`, and achieves a test accuracy of 50%.

Adding:
```python
train_dataset = train_dataset.shuffle()
```
at line 46 seems to solve this issue. Upon retraining the model functions as expected, and achieves a test accuracy of 89.53%.

**Comment by philschmid** [comment]
fixed with #90
